dumper-integration-tests/redshift-tests/src/main/java/com/google/sql/SqlUtil.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2021 Google LLC
```

Comment:
please update

Reply:
I opened a random Java class of Dumper package, copied the licence agreement from there and used it for my classes.
Please, tell me if we need to use any other header.